kong/db/strategies/postgres/connector.lua

```diff
@@ -31,7 +31,7 @@ local fmt          = string.format
 local sub          = string.sub
 local utils_toposort = db_utils.topological_sort
 local insert       = table.insert
-local table_merge  = utils.table_merge
+local table_merge  = kong_table.table_merge
```

Comment:
Could be `local table_merge = require("kong.tools.table").table_merge`?

Reply:
changed